internal/mod/get.go

```diff
@@ -107,5 +136,39 @@ func Download(mod module.Version, fs billy.Filesystem) error {
 		}
 	}
 
+	fmt.Println("why root ", fs.Root())
 	return BillyCopy(fs, mfs)
 }
+
+// Return latest Tag
+func GetLatestTag(r *gogit.Repository, gco *gogit.CloneOptions) (string, error) {
+	var latestTagCommit *object.Commit
+	tags, _ := r.Tags()
+	var latestTagName string
+	err := tags.ForEach(func(t *plumbing.Reference) error {
+		revision := plumbing.Revision(t.Name().String())
+		tagCommitHash, err := r.ResolveRevision(revision)
+		commit, err := r.CommitObject(*tagCommitHash)
+		if err != nil {
+			return err
+		}
+
+		if latestTagCommit == nil {
+			latestTagCommit = commit
+			latestTagName = t.Name().Short()
+			t.Type()
+		}
+
+		if commit.Committer.When.After(latestTagCommit.Committer.When) {
```

Comment:
One short coming of this, if someone is actively working on a repo and decides to push a "dirty" or a "temporary" to test something, it will be caught as the "lastest" by this function because it will be the most recent. Unfortunately, I can't see how we could protect ourselves from that....